docs/user_guide/tensorflow/hops.rst

```diff
@@ -8,59 +8,69 @@ hdfs
 -----------------------
 .. highlight:: python
 
-The ``hdfs`` module provides several ways of interacting with the Hops filesystem, where the Hopsworks project's data is stored. 
+The ``hdfs`` module provides several ways of interacting with the Hops filesystem, where the Hopsworks project's data is stored.
 
-**Pointing to a Dataset in Hopsworks**
+**Getting Project Information**
 
+::
+
+    from hops import hdfs
+    project_user = hdfs.project_user() # returns: ProjectName__ProjectUserId
+    project_name = hdfs.project_name() # returns: ProjectName
+    project_path = hdfs.project_path() # returns: hdfs://ip:port/Projects/ProjectName/
 
-In order to point to a dataset in Hopsworks the ``hdfs.project_path()`` function should be called. The function will return the HDFS path of your project, which is the view shown when clicking at ``Data Sets`` in Hopsworks. To point where your actual data resides in the project, you need to append the full path from there to your Dataset. 
 
-::   
+A common use-case for using the hdfs module to get project information is when you need to provide a path to your project-datasets for reading the dataset in your program, for example using a framework like Spark or Tensorflow. When providing a path to a dataset in your project, you'll typically write ``hdfs.project_path() + DATASET_NAME/...``. What's happening here is that ``hdfs.project_path()`` returns HDFS path to your project, this is the path you preview when you visit the ``Data Sets`` view in the Hopsworks UI. To get the full HDFS path to the dataset, the dataset name is appended to the project path.
 
-    from hops import hdfs
-    my_project_path = hdfs.project_path()
-    
-It is also possible to pass an argument corresponding to a different project for which you want to resolve the path to.    
+By default, ``hdfs.project_path()`` returns the path to the project that you are using while running the code. However, if you want to get the HDFS path to another project, you can pass an argument to the function with the project name:
 
-::   
+::
 
     from hops import hdfs
-    other_project_path = hdfs.project_path('deep-learning-101')    
-   
-    
-A simple use-case might be writing to a file in your project. Depending on whether you are using Python or PySpark kernel, you can follow one of two these different approaches in Jupyter.
+    other_project_path = hdfs.project_path('deep-learning-101') #returns hdfs://ip:port/Projects/deep-learning-101/
```

Comment:
@robzor92 does the path returned include the `ip:port`? I think it shouldn't be there, the hdfs client will handle that as we can have multiple namenodes.

Reply:
Agree. Had alot of issues with this in the past, specifically the error "expecting authority at index 7" which is why it is as it is. Can take a look at it again if it works using hdfs:///

Reply:
tensorflow does not like hdfs:///, it expects hdfs://default/ ref: https://github.com/yahoo/TensorFlowOnSpark/issues/214